module/os/linux/zfs/zfs_ctldir.c

```diff
@@ -977,6 +978,15 @@ zfsctl_snapdir_mkdir(struct inode *dip, char *dirname, vattr_t *vap,
 	return (error);
 }
 
+static void
```

Comment:
Could we add a simple comment as to why we need to do the flush? Thanks.

Reply:
@don-brady Thanks Don. I added comment to the function exportfs_flush(), please take a look.